content/blog/2024/09/kaoto-release-2.2.0/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Kaoto v2.2 release"
+date: 2024-09-20
+draft: false
+authors: [lordrip]
+categories: ["Releases", "Tooling", "Kaoto"]
+preview: "Kaoto 2.2 has been released"
+---
+
+<div style="text-align: center">
+  <img src="cover.png" alt="Kaoto" height="400"/>
+</div>
+
+<br/><strong>
+Following the Apache Camel 4.8 release, we are happy to announce the release of Kaoto 2.2 with many improvements to enhance the user experience.
+
+This time, the theme was to improve the visualizations and the user experience in the Canvas, as well as to provide more flexibility to the users. Read on to discover the key highlights of this release and how they can improve your workflow with Kaoto.
+</strong>
+
+## What’s New in Kaoto 2.2?
+
+### Canvas Improvements
+
+- New look and feel: The flow elements have been redesigned to provide a more modern and clean look. The new design is more intuitive and user-friendly, making it easier to work with your integrations.
+
+<div style="text-align: center">
```

Comment:
I'd avoid adding too much divs around. I think you can use the format like.
```
![Canvas](canvas.png)
_Figure 1. Canvas_
```
I cannot recall if there is the possibility to resize. If not, it would be probably good to publish the image with the required size.

Reply:
ok, I'll reupload the images

Reply:
mmm, the issue with that is the images will lose quality, so they would look fine in the website but if you want to zoom in to see the details, then it will be "_un-pretty_"